src/tests/system/links-orphans-contexts-tags-tasks/expected/references.md

```
@@ -28,8 +43,10 @@
 <details>
 <summary>Show Orphans</summary>
 
-* [Readme][README] `README.md`: [This is a link to a page that doesn't exist - an orphan]
-* [tasks][tasks] `tasks.md`: [Titles] links into [[xxxxxxxxxxxxx]]
+* [Zettel Linter][tasks] `tasks.md`: [Titles] links into [[xxxxxxxxxxxxx]], 
+* [Testing file][links] `links-orphans-contexts-tags.md`: [[https://example.com/author/bob]], [This is a link to a page that doesn't exist - an orphan], 
+* [Readme][README] `README.md`: [[This is a link to a page that doesn't exist - an orphan]], [[wiki]]`, 
+* [Contributing to the project][CONTRIBUTING] `CONTRIBUTING.md`: [TaskCollector](src/collectors/task-collector.ts) - a collector that looks for `[ ], [WikiCollector](src/collectors/wiki-collector.ts) - a collector that looks for `[[WikiLinks]]` and Zettelkasten-style `[yymmddhhmmss]`, [Templator](src/Templator.ts) uses `mustache` to render the output file, using the data collected by the collectors, and some additional annotations to support filtering notes `{{``tag[filter]`, [yyyymmddhhmmss]` links into WikiLinks `[[like-this]]`, 
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements
null

(MD038, no-space-in-code)